packages/oauth-providers/src/providers/linkedIn/index.ts

```diff
@@ -0,0 +1,11 @@
+import type { OAuthVariables } from '../../types'
```

Comment:
Sorry for the confusion. The file name should be all lowercase: `linkedin`, not `linkedIn`.

```
packages/oauth-providers/src/providers/linkedin/index.ts
```

Reply:
Thank you very much!!